bots/pr/src/main/java/org/openjdk/skara/bots/pr/JEPCommand.java

```diff
@@ -55,23 +55,39 @@ private void showHelp(PrintWriter reply) {
                  * `/jep 1234567`
                  * `/jep jep-123`
                  * `/jep JEP-123`
+                 * `/jep 123`
                  * `/jep unneeded`
 
                 Note:
-                The project prefix (`JDK-` in the above examples) is optional if you use an issue ID.
-                The issue type in that case must be `JEP`.
-                The `JEP-` or `jep-` prefix is required if you instead provide a JEP ID.
+                The prefix (`JDK-`, `JEP-` or `jep-` in the above examples) is optional.
+                The bot firstly treats the ID without prefix as a JEP ID.
+                If not found, the bot then treats the ID without prefix as an issue ID.
+                The issue type in any case must be `JEP`.
```

Comment:
```suggestion
                The prefix (i.e. `JDK-`, `JEP-` or `jep-`) is optional. If the argument is given without prefix, it will be tried first as a JEP ID and second as an issue ID. The issue type must be `JEP`.
```

A text block here is good for formatting the syntax and examples, but text should not be forcibly line broken. Consider adding the text paragraph as an additional string so that it may be broken up in code, but not when posted.

Reply:
Fixed.